style/style.css

```diff
@@ -0,0 +1,218 @@
+body {
+    background-color: #ECEEF1;
+    font-family: 'Roboto', sans-serif;
+    color: #4c4c4c;
+}
+main {
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    height: 800px;
+}
+#up-info {
+    display: flex;
+    justify-content: flex-end;
+    text-align: center;
+    float: right;
+    height: 40px;
+    align-items: center;
+    font-size: 12px;
+}
+#up-info a {
+  text-decoration: none;
+  color:#0077c5;
+}
+#up-info button {
+  width: 105px;
+  height: 35px;
+  background-color: #0077c5;
+  color: white;
+  margin-left: 12px;
+  margin-right: 15px;
+  font-family: 'Roboto', sans-serif;
+  font-size: 15px;
+  border: 1px solid #333333;
+  border-radius: 5px;
+  cursor: pointer;
+}
+section {
+  margin-top: 35px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.central-section {
+    display: flex;
+    flex-direction: column;
+    background-color: white;
+    margin-top: 10px;
+    border: 1px solid #c7c7c7;
+    width: 380px;
+    height: 670px;
+    justify-self: center;
+    align-self: center;
+    text-align: center;
+}
+fieldset {
+  border: 0;
+  margin: 0;
+  padding: 0;
+}
+#form h3 {
+  margin-top: 25px;
+  margin-bottom: -15px;
+  font-size: 23px;
+  font-weight: normal;
+}
+#form p {
+  font-size: 14px;
+}
+#form a {
+  text-decoration: none;
+  font-weight: normal;
+  font-size: 12px;
+  color: #0077c5;
+}
+
+.form-info {
+  padding: 20px 20px 0 20px;
+}
+.form-info label p {
+  text-align: left;
+  margin-left: 10px;
+  margin-bottom: -10px;
+  font-size: 12px;
+  font-weight: normal;
+  color: black;
+  margin-bottom: 5px;
+}
+.form-info input {
+  background: #ffffff;
+  border: 1px solid #c6c7cc;
+  padding: 12px;
+  margin-top: 6px;
+  width: 300px;
+  margin-bottom: 30px;
+  border-radius: 3px;
+}
+.form-info input:hover {
+  box-shadow: 0 0 5px 0 rgba(106, 184, 230, 0.774);
+}
+.form-info #opened {
+  width: 323px;
+  height: 44px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border: 1px solid #c6c7cc;
+  border-radius: 3px;
+  margin-left: 8px;
+}
+.form-info #opened:hover {
+  box-shadow: 0 0 5px 0 rgba(106, 184, 230, 0.774);
+}
+.form-info #opened input {
+  border: none;
+  margin: none;
+  height: 19px;
+  margin: 0em;
+}
+.form-info #opened input:hover {
+}
```

Comment:
- [ ] You haven't added styles here so there is no reason to keep this in your file. It can be removed.